__tests__/frontend-proxy.ts

```diff
@@ -73,6 +73,24 @@ describe('handleRequest()', () => {
     })
   })
 
+  describe('prepends language code when FRONTEND_SUPPORT_INTERNATIONALIZATION is "true"', () => {
```

Comment:
Should have test for legacy backend, too (since we don't want to prepend the language code there)